app/src/main/java/com/github/factotum_sdp/factotum/ui/roadbook/RoadBookViewAdapter.kt

```diff
@@ -12,6 +12,7 @@ import com.github.factotum_sdp.factotum.data.DestinationRecord.Action
 import com.github.factotum_sdp.factotum.data.DestinationRecord
 import java.text.SimpleDateFormat
 import java.util.*
+import kotlin.collections.HashMap
 
 /**
  * Adapter for the RecyclerView which will display a dynamic list of DestinationRecord
```

Comment:
Yes I studied the question in the last Sprint, and read on several forum, that ListAdapter had some limitations for implementing a clean Drag and Drop. Hence, I had to use a RecyclerView.Adapter with my own asyncList as attribute, to avoid getting stuck in a dead end.
Next time I will keep the URL of these notes.